mungegithub/submit-queue/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: k8s-submit-queue
+spec:
+  rules:
+  - host: kubernetes.submit-queue.k8s.io
+    http:
+      paths:
+        - backend:
+            serviceName: k8s-kubernetes-sq-status
+            servicePort: 80
+  - host: contrib.submit-queue.k8s.io
+    http:
+      paths:
+        - backend:
+            serviceName: k8s-contrib-sq-status
+            servicePort: 80
+  - host: docs.submit-queue.k8s.io
```

Comment:
docs will be the name of the subdirectory under `mungegithub/submit-queue/deployment/`, which will contain the deployment files for kubernetes.github.io. The next PR adds it. I shortened it to docs rather than using the repo name because I started to hit a 24 character limit on service names when I used the whole string.